draft-ietf-dnsop-svcb-https.md

```diff
@@ -936,7 +936,7 @@ the Internet ("IN") Class {{!RFC1035}}.
 
 All the SvcParamKeys defined in {{keys}} are permitted for use in
 HTTPS RRs.  The default set of ALPN IDs is the single value "http/1.1".
-The "automatically mandatory" keys ({{mandatory}}) are "port", "alpn",
+The "automatically mandatory" keys ({{mandatory}}) are "port"
 and "no-default-alpn".
```

Comment:
Do we want a clarifying note as to what it means for alpn to not be automatically mandatory given that no-default-alpn requires alpn to be present?

Reply:
I don't have any clarification to add on that point.  Suggestions welcome.

Reply:
Added a suggestion above.